<?xml version="1.0"?>
<launch>
     <node name="keyboard_node" pkg="ros_input_output" type="keyboard_node" output="screen" />
</launch>
